src/main/java/com/rcv/Main.java

```diff
@@ -32,11 +31,9 @@
   // param: args command line argument array
   // returns: N/A
   public static void main(String[] args) {
-    try {
-      Logger.setup();
-    } catch (IOException exception) {
-      System.err.print(String.format("Failed to start system logging!\n%s", exception.toString()));
-    }
+    System.out.println("RCVRC Universal Tabulator version 1.0");
```

Comment:
This print statement is unnecessary and makes it say the same thing twice since it says it again after the log is loaded. Please delete.

Reply:
So the thinking here is, the System.out message will be the first thing displayed to users launching from the CLI, before logging is setup.  If there is a logging setup issue at this point, the console output makes more sense because of that line.  The repeated message is added to get the same information into the log file, in the interest of improved forensics / reproducibility.